arches/app/media/css/arches.scss

```diff
@@ -2258,17 +2258,13 @@ div.final-cons-step-separator>h4 {
 
 .workbench-card-sidepanel-header-container {
     border-bottom: 1px solid #ddd;
-    padding: 17px 30px 10px 0px;
+    padding: 10px 20px 10px 5px;
```

Comment:
The padding here seems to be off now; there's less below the bottom border than above; these should be equal.  It also looks like there's a slight shift in alignment to the right; the "x" icon should align with the left side of the border start.

Reply:
Yes you are correct, not sure why this was done previously. Changed back to the previous values.